v2/src/components/appInfoForm/index.tsx

```diff
@@ -33,14 +33,61 @@ type State = {
     netlifyApiRouteUsed: boolean,
     showWebsiteBasePath: boolean,
     showAPIBasePath: boolean
```

Comment:
When both the form and code snippets are visible, the bottom spacing for the form is incorrect. It should use the same spacing as the resubmit state

<img width="764" alt="Screenshot 2022-08-31 at 10 21 36 AM" src="https://user-images.githubusercontent.com/18233774/187605621-8022f610-b2c8-41e6-8920-9b50f673a205.png">

<img width="780" alt="Screenshot 2022-08-31 at 10 22 33 AM" src="https://user-images.githubusercontent.com/18233774/187605637-e9b0dd4f-4ec6-46f8-a063-7309f2ea326e.png">

Reply:
updated